pulp-rpm.spec

```diff
@@ -191,6 +193,8 @@ Summary: The RPM admin client extensions
 Group: Development/Languages
 Requires: pulp-admin-client = %{pulp_version}
 Requires: python-pulp-rpm-common = %{pulp_version}
+Requires: deltarpm
+Requires: python-deltarpm
```

Comment:
I'm not 100% sure, but I need this library in Server plugin and also in admin extension so I think it should be here.